src/__stories__/_utils.ts

```diff
@@ -3,44 +3,44 @@
  */
 import { SizeProp } from '@fortawesome/fontawesome-svg-core';
 
-export const BorderRadius = ['', 'none', 'sm', 'md', 'lg', 'xl', 'full'];
-export const BorderWidth = ['', 'none', 'xs', 'sm', 'md', 'lg'];
-export const BoxShadow = ['', 'sm', 'md', 'lg'];
+export const BorderRadius = ['', '@none', '@sm', '@md', '@lg', '@xl', '@full'];
+export const BorderWidth = ['', '@none', '@xs', '@sm', '@md', '@lg'];
+export const BoxShadow = ['', '@sm', '@md', '@lg'];
 export const TextAlign = ['', 'left', 'right', 'center', 'justify', 'initial', 'inherit'];
 export const OverFlow = ['', 'visible', 'hidden', 'scroll', 'auto', 'initial', 'inherit'];
 export const PositionOpts = ['', 'static', 'relative', 'fixed', 'absolute', 'sticky'];
 
-export const FontSize = ['', 'xs', 'sm', 'md', 'lg', 'xl', '2xl', '3xl', '4xl', '5xl', '6xl'];
-export const FontWeight = ['', 'thin', 'light', 'normal', 'medium', 'semibold', 'bold', 'extrabold'];
+export const FontSize = ['', '@xs', '@sm', '@md', '@lg', '@xl', '@2xl', '@3xl', '@4xl', '@5xl', '@6xl'];
+export const FontWeight = ['', '@thin', '@light', '@normal', '@medium', '@semibold', '@bold', '@extrabold'];
 
 export const Size: SizeProp[] = ['xs', 'lg', 'sm', '1x', '2x', '3x', '4x', '5x', '6x', '7x', '8x', '9x', '10x'];
```

Comment:
Aren't we missing `@` here?

Reply:
Size here refers to the FontAwesome sizes, which are not defined in the theme so don't need the `@`